src/chartjs-plugin-labels.js

```diff
@@ -430,11 +430,14 @@
     return image;
   };
 
+  function isPluginsLabelsDefined(options) {
+    return !!options._context.chart.config._config.options.plugins.labels;
```

Comment:
What happens when .plugins is not actually defined? It can be? I think it throws an error such as "Cannot read property labels of undefined"?

Reply:
I agree with you, I am gonna push a change to account for this case